util-taglib/src/com/liferay/taglib/aui/base/BaseInputTag.java

```diff
@@ -335,6 +339,10 @@ public void setHelpTextCssClass(java.lang.String helpTextCssClass) {
 		_helpTextCssClass = helpTextCssClass;
 	}
 
+	public void setHideTranslationManager(boolean hideTranslationManager) {
+		_hideTranslationManager = hideTranslationManager;
+	}
```

Comment:
To avoid this language awkwardness in getters and setters, it's wise to name a boolean attribute as an adjective, not a verb. In this case, it would be `translationManagerVisible` or `translationManagerHidden`. It also matches the general good practice of last part of name being the most relevant part.

`TranslationManager` is the underlying component name, but component could change in time. We already have `languagesDropdownDirection` as an attribute. Is this referring to the same part of he component? If yes, for consistency, we can call the new attribute something like `languagesDropdownVisible`.